AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -537,6 +537,7 @@ titulky.com#%#//scriptlet("set-constant", "showBanner600", "true")
 @@||thehouseofportable.com^$generichide
 @@||googletagmanager.com/gtm.js$domain=thehouseofportable.com
 @@||googletagmanager.com/gtag/js?id=$domain=thehouseofportable.com
+@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=facciabuco.com
```

Comment:
It won't work on the Safari. You have to move the rule to the end of the file -> `! Common safari_cb_affinity(general)`.